src/libcollectdclient/client.c

```diff
@@ -309,7 +309,7 @@ static int lcc_receive(lcc_connection_t *c, /* {{{ */
     ptr++;
 
   /* Now copy the message. */
-  strncpy(res.message, ptr, sizeof(res.message));
+  strncpy(res.message, ptr, sizeof(res.message) - 1);
   res.message[sizeof(res.message) - 1] = '\0';
```

Comment:
I think that it is reasonable even though this seems to be false-positive alarm.
`res.message[sizeof(res.message) - 1]` is overwritten by zero, so there is no sense to copy anything to this place just one line before.